src/check/canonicalize/CIR.zig

```diff
@@ -94,9 +139,9 @@ fn appendIdentChild(node: *sexpr.Expr, gpa: std.mem.Allocator, ir: *const CIR, n
 
             // Now add the string child directly to the node in its final location
```

Comment:
Should this still mention `child`?

Reply:
resolved in commit d936f9ccc72ad4fdfd76b2debb8147780fae7d2f